ironmon_tracker/GameSettings.lua

```diff
@@ -401,6 +411,10 @@ function GameSettings.setGameVersion(gameversion)
 				versionName = "Pokemon Feuerrote",
 				versionIndex = 6,
 			},
+			[0x00630000] = { -- Japanese
+				versionName = "Pokemon FireRed J",
+				versionIndex = 7,
+			},
```

Comment:
Could be worth mentioning the `v1.0` here? (either directly in `versionName` or as a comment, just so the note is there for anyone else who looks at this)

Reply:
Part of why i left this out is my lack of research to know what rom i was using and what roms are possible. I *think* the rom im using is 1.0. and I *think* its the only one that exist. but im not entirely sure. I dont have a different version that i found that i could test.

Reply:
hm yeah, that's fair i suppose